specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -195,7 +182,14 @@ function TreeView<SCHEMA extends AnyTree>({
         >
           {treeText.editRanks()}
         </Button.Small>
-        <Button.Small onClick={() => setCollapsed(!collapsed)}>
+        <Button.Small
+          disabled={conformation.length === 0}
+          onClick={(e): void => {
+            e.preventDefault;
```

Comment:
First, e.preventDefault is a function. To call it as a function you need to say `e.preventDefault();`
Secondly, you don't need `e.preventDefault();`, as `<Button.Small>` component already takes care of that for you (by verture of adding `type="button"` prop)